frontend/public/components/utils/kebab.tsx

```diff
@@ -108,12 +107,11 @@ export class Kebab extends DropdownMixin {
 
     return <div id={id}>
       { isDisabled ?
-        <Tooltip content="disabled">
-          <div ref={this.dropdownElement} className="co-kebab co-kebab--disabled" >
+        <div ref={this.dropdownElement} className="co-kebab" >
```

Comment:
Since this node is now common to both cases, I recommend moving it outside the ternary.